internal/pkg/template/templates/environment/bootstrap-cf.yml

```
@@ -0,0 +1,28 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+Description: CloudFormation environment template for infrastructure shared among Copilot workloads.
```

nit: can we make the description like this as well:
```suggestion
Description: CloudFormation environment bootstrap template with the necessary roles to create envs and workloads.
```